src/resources/js/aggregator-fields.js

```diff
@@ -68,11 +68,11 @@ tribe_aggregator.fields = {
 			'organizer': ''
 		},
 		detect_type: function ( url ) {
-			if ( ! tribe_aggregator.source_origin_regexp.eventbrite ) {
+			if ( ! ea.source_origin_regexp.eventbrite ) {
 				return null;
 			}
 
-			var baseRegex = tribe_aggregator.source_origin_regexp.eventbrite;
+			var baseRegex = ea.source_origin_regexp.eventbrite;
```

Comment:
no wait, i dont get this ? where is ea defined ?

Reply:
[here](https://github.com/the-events-calendar/the-events-calendar/blob/e6652d59b68ea1d524744b628c9fd744906205b2/src/Tribe/Aggregator/Page.php#L137)

Reply:
Thought - we are passing `tribe_aggregator` into the base function as `ea`, so we should reference it that way